docs/conf.py

```diff
@@ -0,0 +1,69 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+from pathlib import Path
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+def get_version():
+    try:
+        with open(Path(os.path.dirname(
+                os.path.abspath(__file__)), "..", ".version"), "r") as file:
+            return file.read().strip()
+    except Exception:
+        return os.environ.get('GIT_VERSION_NUMBER', 'development')
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Shared Workflows'
+copyright = '2021, Noveto'
```

Comment:
Year should be extracted from `datetime.now().year`

Reply:
Fixed in 717bb165208c0c5f87159669e5c8558577815264